example/index.js

```diff
@@ -21,7 +21,7 @@ import {promisify} from 'node:util';
                         }
                     </style>
                 `,
-                embeddingMode: 'srcdoc',
+                embeddingMode: 'isolated',
```

Comment:
Is this a breaking change? I suggest trying not to change the major version often